src/components/Jobs/components/PreviewsJobsAccordion/PreviewsJobsAccordion.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
```

Comment:
Let's use the next form of notation for this.

```suggestion
import get from 'lodash/get';
```